pkg/subctl/cmd/logs.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+)
+
+func getConnectivityPodsLogs(clientSet kubernetes.Interface) (bool, error){
+	pods, err := network.FindPods(clientSet, v1.ListOptions{LabelSelector: "app=submariner-gateway"})
```

Comment:
Don't we also want to collect the route-agent logs? There's also globalnet and the network plugin syncer.

Reply:
THe logic would be same for those pods. I will add those pods once this is confirmed as the logic will get repeated